packages/divider/src/divider.vue

```diff
@@ -0,0 +1,34 @@
+<template function>
+  <div v-bind="$attrs" :class="['el-divider', `el-divider--${direction}`]">
```

Comment:
`v-bind="$attrs"` dose not need in vue-next. See [active-rfcs/0031-attr-fallthrough.md](https://github.com/vuejs/rfcs/blob/master/active-rfcs/0031-attr-fallthrough.md)

`<template function>` dose `function` necessary?